src/store/state.ts

```diff
@@ -143,7 +151,15 @@ export const stateModel: StateModelIF = {
     userCancelledAnalysis: false,
     waitingAddressSearch: null,
     isLoadingSubmission: false,
-    hotjarUserId: ''
+    hotjarUserId: '',
+    businessLookup: {
+      identifier: '',
+      legalType: CorpTypeCd.BC_CCC,
+      bn: '',
+      status: EntityStates.ACTIVE,
+      name: '',
+      disabled: false
+    }
```

Comment:
This looks too much like real data. Is there an "empty business lookup" const you could use here instead? Or could the whole object be set to null?

Reply:
I set to null